imports/ui/explorer-components/item-list.jsx

```diff
@@ -52,7 +52,7 @@ export function CaseList ({ allCases, onItemClick }) {
   )
 }
 
-CaseList.propTypes = {
-  allCases: PropTypes.array.isRequired,
+ItemList.propTypes = {
```

Comment:
I think there's very little in common between items of type "case" and items of type "report". Many of the things here relate to case items only. It might be cleaner to have a separate component for each type.